metricbeat/docs/modules_list.asciidoc

```diff
@@ -189,6 +192,7 @@ This file is generated! See scripts/mage/docs_collector.go
 include::modules/aerospike.asciidoc[]
 include::modules/apache.asciidoc[]
 include::modules/aws.asciidoc[]
+include::modules/beat.asciidoc[]
```

Comment:
This file is missing. You gotta run `make update` to generate it.

Reply:
Oops! Added in 6a8cc8ae2.